interactive-clap-derive/src/derives/interactive_clap/common_methods/choose_variant.rs

```diff
@@ -29,16 +32,21 @@ pub fn fn_choose_variant(
                     }
                 }
             };
+            dbg_cond!(attr.path.is_ident("strum_discriminants"));
             if attr.path.is_ident("strum_discriminants") {
                 for attr_token in attr.tokens.clone() {
                     if let proc_macro2::TokenTree::Group(group) = attr_token {
-                        if &group.stream().to_string() == "derive(EnumMessage, EnumIter)" {
+                        let group_stream_no_whitespace =
+                            group.stream().to_string().replace(" ", "");
```

Comment:
maybe split_whitespaces and then collect, or something like that. 
maybe some weirdo uses tabs or something like that and ignores fmt.

Reply:
it actually wasn't dependent on presence of whitespace in original source code input, but rather defined
by https://docs.rs/proc-macro2/latest/proc_macro2/struct.TokenStream.html#impl-Display-for-TokenStream

the only weirdo who could change this to using tabs would be https://crates.io/users/dtolnay, which is unlikely

Reply:
resolved by https://github.com/near-cli-rs/interactive-clap/pull/26/commits/9bedd978f3bc377ae518335723ae4e8f74e0c978